feature/security/gnsi/certz/server_certificate_rotation/README.md

```diff
@@ -0,0 +1,97 @@
+# X.X: Server Certificate Rotation
+
+## Summary
+
+Certificates on network devices (servers) must be rotated over time for various
+operational reasons. The ability to perform a rotation is a key component of
+safe operation practices.
+
+## Baseline Setup
+
+### Input Args
+
+   * the set of certificate testdata generated with the mk_cas.sh script
+   in featureprofiles/feature/certificate-authorities/
+
+### DIT Service Setup
+
+Configure the DUT to enable the following services (that are using gRPC) are up
+and required user mTLS for authentication:
+
+   * gNMI
+   * gNOI
+   * gNSI
+   * gRIBI
+
+Be prepared to load the relevant trust_bundle.pem file for each test Certificate
+Authority(CA) under test on the DUT. Each CA has an RSA and ECDSA form, both
+must be tested.
+
+## Tests
+
+### Certz-3.0
+
+Perform these positive tests:
+
+Test that a server certificate can be rotated by using the gNSI certz api if
+the certificate is requested without the device generated CSR.
+
+Perform this test with both the RSA and ECDSA types.
+
+   0) Build the test data, configure the DUT to use the ca-0001 form
+      key/certificate/trust_bundle, use the server-${TYPE}-a key/certificate.
+
+   1) With the server running, connect and note that the ceritficate loaded
+      is the appropriate one.
+
+   2) Use the gNSI Rotate RPC to load a server-${TYPE}-b key and certificate
+      on to the server.
+
+   3) Test that the certificate is properly loaded, using the Probe RPC.
+      Note that the new certificate is properly served by the server.
+
+   4) Send the Finalize RPC to the server.
+
+   5) Verify that the server is now serving the certifcate properly.
```

Comment:
Any suggestion about how to validate the -b suite are in effect?

Reply:
added text.